### ISSUE TITLE
Update color bug after blocking/unblocking work profile

### ISSUE THREAD

**Issue by kohanla** [MuntashirAkon/AppManager#849]
- [x] I know what my device, OS and App Manager versions are
- [ ] I know how to take logs
- [x] I know how to reproduce the issue which may not be specific to my device

**Describe the bug**

V3.0.0

If an app contains unblocked trackers the labels is colored orange. After blocking all, the label automatically turns green. After unlblocking at least one the label turns orange.

But this does not happen when the app comes from a work profile. In that case the user has to refresh the app view to update the color.

This is  not a critical isulsue of course. Just FYI.

**Comment by MuntashirAkon** [comment]
> But this does not happen when the app comes from a work profile. In that case the user has to refresh the app view to update the color.

When a component of an app is modified, it sends a broadcast message to App Manager (if it is running) which allows it to update the page. But for an app on the work profile, it cannot receive any broadcast message and, therefore, unable to update the page. This could be fixed, of course. But as you've said it yourself, this is a low priority (and a cosmetic) issue and shall be fixed after finishing all the backlogs.

**Comment by MuntashirAkon** [comment]
Low priority issues are still _opened_ issues. We have other tracking methods via labels.
